src/LBFactory.sol

```diff
@@ -45,7 +47,7 @@ contract LBFactory is Ownable2Step, AccessControl, ILBFactory {
     address private _feeRecipient;
     uint256 private _flashLoanFee;
 
-    address private _lbPairImplementation;
+    address private _lbPairBeacon;
```

Comment:
i recommend to make it immutable because it is not going to change and move assignment to the constructor to save on reading from the storage on every call
```suggestion
    address private immutable _lbPairBeacon;
```

Reply:
Making the _lbPairBeacon immutable will require it to be set in the constructor level, which:
1. Raced condition with the `lbPairUpgradeableBeacon` since it also required the factory contract to be set in the constructor level.

2. Will require refactoring with the unit tests

Reply:
ok
makes sense
besides both are not frequently used, so it's not a big deal in effect compared to the required efforts. 
thanks